plugins/catalog-backend/src/api/deprecatedResult.ts

```diff
@@ -68,9 +68,9 @@ export function generalError(
  */
 export function location(
   newLocation: LocationSpec,
-  optional?: boolean,
+  _optional?: boolean,
```

Comment:
left this in but silencing the linter, the entire function is deprecated and will go away so let's do things step by step